avocado/core/data_dir.py

```diff
@@ -212,7 +220,7 @@ def __del__(self):
 _tmp_tracker = _TmpDirTracker()
 
 
-def get_tmp_dir():
+def get_tmp_dir(basedir=None, keep_tmp=False):
```

Comment:
`keep_tmp=False` seems more clear about the `keep_tmp` usage and intention. Why it would be better to use `keep_tmp=None`?